setup.py

```diff
@@ -47,7 +47,7 @@ def run(self):
     packages=find_packages(exclude=['ez_setup', 'examples', 'tests']),
     include_package_data=True,
     zip_safe=False,
-    install_requires=["gevent>=1.0.1", "flask>=0.10.1", "requests>=2.4.1", "msgpack-python>=0.4.2"],
+    install_requires=["gevent>=1.0.1", "flask>=0.10.1", "requests>=2.4.1", "msgpack-python>=0.4.2", "future>=0.15.0", "six>=1.9.0"],
```

Comment:
I don't think gevent 1.0.1 is compatible with Python 3. You need >=1.1, see https://github.com/gevent/gevent/blob/master/changelog.rst#11a1-jun-29-2015.